Models/Base/MetaData/POCO/Segment.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.Base.MetaData.POCO
+{
+    public class Segment
+    {
+        [JsonPropertyName("name")] public string Name { get; set; }
+
+        [JsonPropertyName("value")] public List<Value> Value { get; set; }
```

Comment:
> __[FxCop]__ Change 'List<Value>' in 'Segment.Value' to use 'Collection<T>', 'ReadOnlyCollection<T>' or 'KeyedCollection<K,V>' ([view](https://sider.review/gh/repos/336560435/pulls/56#issue-5402037))
>
> <details>
>
> | Rule | Severity | Category |
> | ---- | -------- | -------- |
> | `CA1002` | `Warning` | Design |
>
> References:
> - <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1002>
>
> You can [close this issue](https://sider.review/gh/repos/336560435/pulls/56#issue-5402037) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Most of the rest of the code uses `Collection`. [Difference](https://stackoverflow.com/questions/398903/what-is-the-difference-between-list-of-t-and-collectionof-t). @kpaubert knows more about this.

Reply:
Changed know, using the json2csharp converter it always chooses  `List`